### PR TITLE
Implement NPS survey

### PR DESCRIPTION
**Pull request by davidangb**
early PR to get automation tests running.

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by davidangb** [comment]
PR run had 6 failures: https://fc-jenkins.dsp-techops.broadinstitute.org/job/swatomation-pipeline/7997/
```
[info] *** 6 TESTS FAILED ***
[error] Failed: Total 125, Failed 6, Errors 0, Passed 119, Ignored 4
[error] Failed tests:
[error] 	org.broadinstitute.dsde.firecloud.test.previewmodal.PreviewSpec
[error] 	org.broadinstitute.dsde.firecloud.test.analysis.MethodConfigSpec
[error] 	org.broadinstitute.dsde.firecloud.test.methodrepo.MethodRepoSpec
```

all have errors related to the surveymonkey overlay:
```
[info] - Preview Modal should display correct message when file not previewable for dos:// link *** FAILED *** (1 minute, 44 seconds)
[info]   org.openqa.selenium.WebDriverException: unknown error: Element <a href="..." class="ltr-marked">dos://broad-dsp-dos.storage.googleapis.com/dos.json</a> is not clickable at point (867, 984). Other element would receive the click: <div smcx-modal-headline="" class="smcx-modal-title">...</div>

[info] - populates connection inputs *** FAILED *** (38 seconds, 949 milliseconds)
[info]   org.openqa.selenium.WebDriverException: unknown error: Element <a href="blob:https://firecloud-fiab.dsde-qa.broadinstitute.org:22443/e08f6248-8c57-406f-9a39-babbaeb15d0b" download="inputs.json" data-test-id="download-link" style="text-decoration: none; color: rgb(69, 127, 210);">...</a> is not clickable at point (734, 997). Other element would receive the click: <iframe width="100%" height="100%" frameborder="0" allowtransparency="true" src="https://www.surveymonkey.com/r/NWBMYFH"></iframe>
```

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/UserFixtures.scala
Comment (davidangb) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/UserFixtures.scala]: does this need a "wait until modal is gone" step?

Comment (ghost) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/UserFixtures.scala]: added!

**Comment by davidangb** [comment]
:+1: for code changes! I can't approve this PR since I started it; please consider this an approval for the last two commits

still left to do before merging:
- [x] Joe to code-review the first two commits of this PR
- [x] see if you can find any documentation on when the survey appears/does not appear for a given user, as this may affect when we see it in tests
- [x] work with Ilyana to ensure that survey results are being collected and can be seen in surveymonkey
- [x] share knowledge about this feature & test changes with QA and/or with all of workbench
- [x] get ok from Ilyana that the survey is ready to release

**Review by ghost — APPROVED** [review]
👍

**Comment by ghost** [comment]
Tiffany has confirmed that SurveyMonkey is getting survey responses from my manual tests. According to Tiffany, SurveyMonkey uses a cookie to prevent the survey from appearing if someone has already answered or declined a survey. If someone responds to or declines the survey and clears their cookies or visits the site from a browser that blocks cookies, they'll see the survey again.

**Pull request merged**
